torchci/lib/bot/verifyDisableTestIssueBot.ts

```diff
@@ -101,18 +119,13 @@ export function formValidationComment(
   username: string,
   authorized: boolean,
   testName: string,
-  platforms: [Set<string>, Set<string>]
+  platformsToSkip: string[],
+  invalidPlatforms: string[]
```

Comment:
Q: Shy shouldn't they be `Set<string>`? as fundamentally same platform can not be skipped more than once (or be invalid more than once)

Reply:
I changed the function that retrieves them (parseBody) do the set -> list conversion inside it since all the use cases changed them to string lists almost immediately after getting them, should I change it back?